src/frontend/Player/createVideojsPlayer.ts

```diff
@@ -187,6 +189,15 @@ export const createVideojsPlayer = (
   /**************** Interacted event *************************/
   const interacted = (qualityLevels?: QualityLevels): void => {
     if (!isInitialized) {
+      // For a live video, no event to detect when the video is fully initialized is triggered
+      // before the first "play" action. To mitigate this, we can call "initialize"
+      // on the first "interact" action and abort the interact itself. The first interact action is
+      // when the first quality to play is chosen, the default one. To choose it, all quality
+      // available must be read in the HLS manifest. So we can consider at this time that the
+      // video is initialized.
+      if (video.live_state) {
+        initialize();
```

Comment:
I find the commit message a bit hard to understand from the outside. Maybe we could add more context and try to simplify it.

```suggestion
      // At the moment, we don't have any XAPI event that detects when the video is fully loaded,
      //It means, that if there is no "play" action, we don't currently log this interaction. 
      //In order to fix this and catch this event, we can call "initialize"
      // on the first "interact" action and "abort the interact itself." => sorry "abort the interact itself. " I don't really understand this point 
      We can consider that the video is initialized when all formats to play the video are ready to be chosen.
      
      if (video.live_state) {
        initialize();
```

Reply:
It's not  XAPI events, but video events.
`abort the interact itself` means we stop the interact action, we don't log it. 
I think it's a technical subject, we need technical word to explain it.

Reply:
So I suggfest to change `abort the interact itself` by `and we don't log this interaction`. WDYT ?

Reply:
ok